### PR TITLE
Feature/24294 data import utility

### PR DESCRIPTION
**Pull request by istherepie**
- [ ] Have you updated the documentation?
- [x] Have you performed a smoke test of the application?
- [ ] Have you written tests for your changes?
- [ ] Have you updated the release notes?

== Preparation for PR ==
** NOTE** This branch is NOT yet completed

Please note that several things are missing from in this branch. 
 - the actual import (HTTP) functionality
 - support for authentication
 - docstrings

API documentation and tests are in the works!

### CODE REVIEW

File: tooling/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Development
```
Comment (danchr) [inline tooling/.gitignore]: Adding a `tooling` directory is inappropriate, in my opinion. The importer is part of MO, and should use the MO codebase where it makes sense. As such, this should be in the `backend` directory, and preferably as part of the `mora` package — likely as a subpackage.

File: .gitignore [hunk 2]
```diff
@@ -84,6 +84,7 @@ celerybeat-schedule
 /venv/
 /backend/venv-*/
 /backend/venv/
+venv-*
```
Comment (danchr) [inline .gitignore]: We should no longer create such directories — we now just use `venv`.

File: tests/fixtures/importing/create_dummy_fixtures/dummy_data_creator.py [hunk 3]
```diff
@@ -0,0 +1,376 @@
+""" Create dummy data to populate MO """
```
Comment (danchr) [inline tests/fixtures/importing/create_dummy_fixtures/dummy_data_creator.py]: I see this is a left-over from @robertjensen's changes — although it's a useful POC we should delete if it isn't going to get used.

File: tooling/os2mo_data_import/setup.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# -- coding: utf-8 --
+
+from setuptools import setup
+
+setup(
+    name="os2mo-importer",
+    version="0.0.1",
+    description="Data import utility for os2mo",
+    author="Steffen Park",
```
Comment (danchr) [inline tooling/os2mo_data_import/setup.py]: `"Magenta ApS"`

File: tooling/os2mo_data_import/setup.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    author_email="steffen@magenta.dk",
```
Comment (danchr) [inline tooling/os2mo_data_import/setup.py]: `"info@magenta.dk"`

**Comment by istherepie** [comment]
Only relevant code cherry picked - this should fully resolve the duplication.

**Comment by istherepie** [comment]
Closing due to the need for further refactoring.

**Pull request closed without merging**